src/layman/upgrade/upgrade_v2_0.py

```diff
@@ -568,6 +589,12 @@ def delete_old_workspaces():
             gs_util.delete_db_store(gs_workspace, auth=settings.LAYMAN_GS_AUTH, store_name='postgresql')
             gs_util.delete_workspace(gs_workspace, auth=settings.LAYMAN_GS_AUTH)
 
+        # db
+        statement = f'''
+        DROP SCHEMA IF EXISTS {workspace};
```

Comment:
Add tests, that schemas do not exists after migration.

Reply:
Implemented

Reply:
Implemented